mobile/src/main/java/info/hossainkhan/dailynewsheadlines/HealinesPresenter.java

```diff
@@ -64,19 +62,19 @@ public void loadHeadlines(final boolean forceUpdate) {
                 .subscribe(new Subscriber<InlineResponse200>() {
                     @Override
                     public void onCompleted() {
-                        Log.d(TAG, "onCompleted() called");
+                        Timber.d(TAG, "onCompleted() called");
```

Comment:
Note, `Timber.d(TAG, "onCompleted() called");` is **wrong arguments**

It should be `Timber.d("onCompleted() called");` instead **without any `TAG`** ... this was fixed later in the project.